ts/store/reducers/onboarding.ts

```diff
@@ -32,17 +29,17 @@ const reducer = (
   action: Action
 ): OnboardingState => {
   switch (action.type) {
-    case getType(tosAccept.success):
-      return {
-        ...state,
-        isTosAccepted: true
-      };
     case getType(fingerprintAcknowledge.success):
       return {
         ...state,
         isFingerprintAcknowledged: true
       };
 
+    case getType(tosAccepted):
```

Comment:
Why you need this?

Reply:
I thought it was required even if the app has no payload. Removed within https://github.com/teamdigitale/io-app/pull/1047/commits/ff144f5804d638892b5d112af35b8387b39d33e1